### PR TITLE
#164190681 Add Path Resolver

### PR DESCRIPTION
**Pull request by mrkingz**
#### What does this PR do?
- add path resolver
 #### Description of Task to be completed?
- Create aliases for directories and register custom module paths
 #### What are the relevant pivotal tracker stories?
- [#164190681](https://www.pivotaltracker.com/n/projects/2240406/stories/164190681)

### CODE REVIEW

**Review by eokwukwe — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "@sendgrid/mail": "^6.3.1",
     "bcrypt": "^3.0.4",
     "cors": "^2.8.4",
+    "crypto-js": "^3.1.9-1",
```
Comment (obasajujoshua31) [inline package.json]: please remove crypto-js.

File: server/emails/constants/index.js [hunk 2]
```diff
@@ -1,3 +1,17 @@
+const verifyIntro = `Welcome to Authors Haven!
```
Comment (benfluleck) [inline server/emails/constants/index.js]: all these are constant please change to uppercase.

**Review by benfluleck — CHANGES_REQUESTED** [review]

File: server/helpers/jwt.js [hunk 3]
```diff
@@ -1,35 +1,44 @@
+/* eslint-disable import/no-unresolved */
```
Comment (benfluleck) [inline server/helpers/jwt.js]: Why is this here?

File: server/models/users.js [hunk 4]
```diff
@@ -169,24 +164,12 @@ export default (sequelize, Sequelize) => {
...
-    this.save;
-    await this.reload;
+    Object.keys(user).forEach(key => {
+      this[key] = ["firstName", "lastName"].includes(key)
+        ? user[key] || this[key]
+        : user[key];
+    });
+    this.save();
     return this;
   };
```
Comment (obasajujoshua31) [inline server/models/users.js:174]: are you sure this action is saving?, you are not reloading it with await this.reload()?  before return this

File: server/tests/models/notifications.spec.js [hunk 5]
```diff
@@ -1,11 +1,10 @@
...
 import chaiAsPromise from "chai-as-promised";
-import models from "../../models";
-import { userData } from "../fixtures/models/userData";
-import { articleData } from "../fixtures/models/articleData";
+import database from "@models";
+import { userData, articleData } from "@fixtures";
 
 chai.use(chaiAsPromise);
-const { Users, Articles, Notifications } = models;
+const { Users, Articles, Notifications } = database;
```
Comment (obasajujoshua31) [inline server/tests/models/notifications.spec.js]: change the database to models please.

File: server/helpers/jwt.js (hunk 3 above)
Comment (mrkingz) [reply server/helpers/jwt.js]: Because this is having a redline even though it resolves appropriately, and I don't disable this rule for the entire project; hence, I didn't add it to Eslint rules

File: server/models/users.js (hunk 4 above)
Comment (mrkingz) [reply server/models/users.js:174]: This already has the update, no need to reload it.

File: server/routes/index.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import { Router } from "express";
-import authRouter from "./auth";
-import userRoutes from "./userRoutes";
-import articleRoutes from "./article";
+import { authsRouter } from "@routes/auths";
```
Comment (benfluleck) [inline server/routes/index.js]: authRouter what is auths 🤔. Is it not only one router?

File: server/routes/index.js [hunk 7]
```diff
@@ -1,7 +1,7 @@
...
+import { usersRouter } from "@routes/users";
+import { articlesRouter } from "@routes/articles";
```
Comment (benfluleck) [inline server/routes/index.js]: Same with articles, IT is only one Router na

**Review by benfluleck — CHANGES_REQUESTED** [review]

**Review by benfluleck — APPROVED** [review]
